### ISSUE TITLE
Parallel bundle download & decompression 

### ISSUE THREAD

**Issue by matejvasek** [crc-org/crc#4336]
I think it might be slightly faster if bundle download and decompression would be done in parallel. IIUIC right now bundle is first downloaded and only then decompressed.

/kind enhancement

**Comment by cfergeau** [comment]
Yep, interesting idea to try, as the bundle download is mainly limited by network speed, while decompression is CPU-bound. Should save some time at the end of download.

**Comment by praveenkumar** [comment]
@vyasgun do you want to take a look into this?

**Comment by vyasgun** [comment]
@praveenkumar yes, assigned to myself.
